lib/transport/abstract-transport.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var util = require("util");
+var EventEmitter = require('events');
```

Comment:
Oh, hey, looking at latest docs looks like that's ok now. Not sure how well it does on previous versions, though. @mbroadst - any ideas when .EventEmitter stopped being required? Looks like after 0.10 maybe?